packages/instrumentation-document-load/src/instrumentation.ts

```diff
@@ -41,10 +41,7 @@ import {
 import { AttributeNames } from './enums/AttributeNames';
 /** @knipignore */
 import { PACKAGE_NAME, PACKAGE_VERSION } from './version';
-import {
-  SEMATTRS_HTTP_URL,
-  SEMATTRS_HTTP_USER_AGENT,
-} from '@opentelemetry/semantic-conventions';
```

Comment:
looks like you can remove the dependency from package.json for the sem conv package